user_sync/rules.py

```diff
@@ -617,8 +679,16 @@ def write_remove_list(self, file_path, dashboard_users):
                 writer.writerow({'user': username, 'domain': domain})
                 total_users += 1
         self.logger.info('Total users in remove list: %d', total_users)
-                
-        
+
+    def log_after_mapping_hook_scope(self, before_call):
```

Comment:
Agree with your complaint, though I can't say I'm crazy about either proposed fix. Leaving alone for now.